### PR TITLE
Repo Migration - Replaced all references to timehop by adsbynimbus

### PR DESCRIPTION
**Pull request by brugg**

### CODE REVIEW

File: .github/workflows/kotlin.yml [hunk 1]
```diff
@@ -66,14 +66,14 @@ jobs:
...
         with:
           role-to-assume: ${{ vars.AWS_ROLE }}
           aws-region: us-east-1
 
       - name: Publish
-        if: github.repository == 'timehop/nimbus-openrtb' && github.event_name == 'release'
+        if: github.repository == 'adsbynimbus/nimbus-openrtb' && github.event_name == 'release'
         env:
           ORG_GRADLE_PROJECT_version: ${{ steps.tag_name.outputs.VERSION }}
         run: ./gradlew publish
```
Comment (marcsantiago) [inline .github/workflows/kotlin.yml:79]: @brugg confirm with Jason. I don't know much about the Kotlin stuff

File: README.md [hunk 2]
```diff
@@ -6,10 +6,10 @@ Platform specific implementations of Nimbus OpenRTB Request and Response models
...
 ### Supported Platforms
 
-- [![C#](https://github.com/timehop/nimbus-openrtb/actions/workflows/csharp.yml/badge.svg)](csharp)
-- [![Go](https://github.com/timehop/nimbus-openrtb/actions/workflows/go.yml/badge.svg)](go)
-- [![Kotlin](https://github.com/timehop/nimbus-openrtb/actions/workflows/kotlin.yml/badge.svg)](kotlin)
-- [S2S](https://github.com/timehop/nimbus-openrtb/wiki/Nimbus-S2S-Documentation)
+- [![C#](https://github.com/adsbynimbus/nimbus-openrtb/actions/workflows/csharp.yml/badge.svg)](csharp)
+- [![Go](https://github.com/adsbynimbus/nimbus-openrtb/actions/workflows/go.yml/badge.svg)](go)
+- [![Kotlin](https://github.com/adsbynimbus/nimbus-openrtb/actions/workflows/kotlin.yml/badge.svg)](kotlin)
+- [S2S](https://github.com/adsbynimbus/nimbus-openrtb/wiki/Nimbus-S2S-Documentation)
```
Comment (marcsantiago) [inline README.md:12]: Did this package actually get migrated to adsbynimbus org? If it's not you can't edit any go code.

**Review by marcsantiago — CHANGES_REQUESTED** [review]
Go code needs to be migrated to the Org before any code changes or compilation errors will happen.

File: README.md (hunk 2 above)
Comment (brugg) [reply README.md:12]: According to what I saw everything will be migrated.

File: .github/workflows/kotlin.yml (hunk 1 above)
Comment (brugg) [reply .github/workflows/kotlin.yml:79]: Sure. @JasonSznol can you take a look

**Review by renam — APPROVED** [review]

**Pull request closed without merging**